docs/data/toolpad/core/components/account/account.md

```diff
@@ -30,16 +30,45 @@ When signed out, the component renders as an inline sign in button within the da
 
 ## Customization
 
+The `<Account />` component is composed of multiple sub-components:
+
+- `<SignInButton />`
+- `<AccountPreview />`
+- `<Popover />`
+- `<SignOutButton />`
+
+You can pass extra props to them through the `slotProps` prop on the `<Account />` component. You may also completely override them by passing your own custom components to `<Account />` through the `slots` prop.
+
+The labels can be customized through the `localeText` prop.
+
 ### Slot Props
 
-The underlying `signInButton`, `signOutButton` and `iconButton` components can be customized by passing in `slotProps` to the `Account` component.
+The `AccountPreview` component has two variants, `condensed` (the default) and `expanded`. You can change the variant used inside `<Account />` by passing in custom props through `slotProps`:
 
-Labels for the sign in and sign out buttons can be customized through the `localeText` prop.
+{{"demo": "AccountCustomSlotProps.js", "bg": "outlined" }}
 
-{{"demo": "AccountCustom.js", "bg": "outlined" }}
+### Labels
+
+You can pass in custom labels – including of different languages – using the `localeText` prop.
+
+{{"demo": "AccountCustomLocaleText.js", "bg": "outlined" }}
 
 ### Slots
 
-You can pass in your own items to the `Account` menu through the `menuItems` slot to add additional menu items in the space between the user's account details and the sign out button, to create larger, more complex menus:
+You can pass in your own components inside the `Account` popover through the `slots` prop.
+
+#### Popover Content
+
+You can wrap the default sub-components – such as `SignOutButton` and `AccountPreview` – and wrap them in `AccountPopoverHeader` and `AccountPopoverFooter` components to create custom account popovers, as shown in the following demos:
+
+##### Crypto Wallet
+
+You can wrap the default `SignOutButton` with your own action buttons inside `AccountPopoverFooter` to customize the popover footer section:
+
+{{"demo": "AccountSlotsWallet.js", "bg": "outlined" }}
+
+##### Account Switcher
```

Comment:
Perhaps we could show the account switcher before the crypto wallet? The switcher feels a bit less complex visually.